packages/web/src/components/CheckboxField/_CheckboxField.scss

```diff
@@ -0,0 +1,81 @@
+@use 'theme';
+@use '../../tools/typography';
+@use '../../tools/accessibility';
+
+.CheckboxField {
+    display: inline-flex;
+    min-width: 0;
```

Comment:
Do we need this? If not, please remove it.

Reply:
Fixed.